python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -52,8 +52,8 @@ def makeGangaListByRef(_list, preparable=False):
     result = GangaList()
     if len(_list) == 0:
         return result
-    temp_list = [stripProxy(element) for element in _list]
-    result._list = temp_list
+
+    result._list.extend([stripProxy(element) for element in _list])
```

Comment:
So I think we're probably in agreement that makeGangaListByRef should go back to just being a ref copy and makeGangaList should be (essentially) a deepcopy and be used mostly unless the reference version is specifically needed. Is that the case?

If so, I'll start working on that but before then it would be very good to get a performance test in to make sure we don't cripple the performance improvements already in. Has anyone got a good example of this they can put in with a comparison on, say 6.0.44/6.1.x?